src/ProjectionUtil/ProjectionUtil.js

```diff
@@ -109,8 +110,8 @@ export class ProjectionUtil {
    * @return {string} Converted value.
    */
   static toDms(value) {
-    const deg = parseInt(value, 10);
-    const min = parseInt((value - deg) * 60, 10);
+    const deg = Math.floor(value);
+    const min = Math.floor((value - deg) * 60);
```

Comment:
Are we confident, that this does not change behaviour? This is well tested, isn't it?

Reply:
This does not change behavior, but the documentation is contradictory here. On the one hand it says
> Converts geographic coordinates given in DDD format like `DD.DDDD°`
But on the other hand it says the function only accepts numbers.

The old code for `toDmm` would throw errors if it was passed a string, the test was only there for numbers.
```
  // assume value is '123.123⁰'
  const deg = parseInt(value, 10); // this works
  const min = parseInt((value - deg) * 60, 10); // this works
  const sec = ((value - deg - min / 60) * 3600); // this will throw
```

If we would do something like this before it would work for both strings and numbers.
```
const converted = _isString(value) ? parseFloat(value, 10)
```

Reply:
The intention of using `parseInt` here was actually the one of flooring the value. There could be sideeffects (like what would  `parseInt` do to a number `0.123e+123`? it should be converted to a string and the back to an integer). I assume `Math.floor` is the correct function here.

Reply:
Fine for me, thanks again for the rationale.